src/analyse.ts

```diff
@@ -21,8 +21,8 @@ namespace Analyse {
     }
 
     export async function analyse(
-        manifestPath: string, analysisReportName: string, failOn: string
-    ): Promise<void> {
+        manifestPath: string, analysisReportName: string,
+    ): Promise<string> {
```

Comment:
you could have this return `"none" | "warning" | "error"`

Reply:
Okay